getting-started/vercel.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Vercel
+description: Implementing Edgee in front of Vercel for better performance and SEO control
+icon: stairs
+---
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+Edgee can act as a proxy in front of your Vercel project to **improve performance**, **enhance security**, and **simplify integrations**.
+
+This guide explains two different implementation options depending on your needs:
+
+- **Standard Setup** → Recommended if SEO is **not** a priority and you simply want Edgee to proxy requests for performance or security.
+- **SEO-Optimized Setup** → Recommended when SEO is important and you want search engines to **only index your custom domain**, not the default `.vercel.app` domain.
+
+### How These Setups Differ
+
+The key difference between the two approaches is **how requests to your `.vercel.app` domain are handled**:
+
+- In the **Standard Setup**, requests to your `.vercel.app` domain are **not redirected** to your custom domain.  
+  This setup is simpler but can result in **duplicate content** being indexed by search engines, which may negatively affect SEO.
+- In the **SEO-Optimized Setup**, a **301 redirect** ensures that any request to `.vercel.app` is automatically redirected to your custom domain.  
+  This consolidates traffic and search rankings under a **single, branded domain**, which is best for SEO.
+
+---
+
+## Standard Setup Overview
+
+Use this setup when you only need Edgee in front of Vercel for performance or security, and SEO isn’t a concern.
+
+Integrating Edgee with Vercel involves three steps:
+
+1. Get your Vercel project domain  
+2. Configure Edgee to point to your Vercel domain  
+3. Ensure routing settings are correctly configured
+
+
+### Step 1 — Get Your Vercel Domain
+
+1. Log in to your Vercel dashboard.
+2. Copy your project’s default domain (e.g., `myapp.vercel.app` or `nextjs-boilerplate-abc1234.vercel.app`).
+
+   <img src="/images/vercel/verceldomain.png" />
+
+
+### Step 2 — Configure Edgee Routing
+
+1. Log in to Edgee.
+2. Go to **Project Settings → Routing**.
+3. Create a new route:
+   - **Origin URL** → your Vercel project domain from Step 1
+   - **Host Override** → set this to your Vercel domain (e.g. `nextjs-boilerplate-abc1234.vercel.app`)
+   - **SNI** → set this to your Vercel domain  
```

Comment:
are these 3 using the very same value? if so, we are describing it in 3 different ways (and that's quite confusing)

Reply:
I mean, let's just say it simply: "Use the same value for Origin URL, Host OVerride, SNI, and Cert Hostname.